packages/plugin-rsc/e2e/browser-mode.test.ts

```diff
@@ -2,14 +2,23 @@ import { expect, test, type Page } from '@playwright/test'
 import { useFixture } from './fixture'
 import { defineStarterTest } from './starter'
 
-test.describe('dev-browser-mode', () => {
-  // Webkit fails by
-  // > TypeError: ReadableByteStreamController is not implemented
-  test.skip(({ browserName }) => browserName === 'webkit')
+// Webkit fails by
+// > TypeError: ReadableByteStreamController is not implemented
+test.skip(({ browserName }) => browserName === 'webkit')
```

Comment:
@kasperpeulen FYI, I'm not sure what to do about this, but probably this means this setup would be broken on all Safari. I tried `web-streams-polyfill` but it probably broke other things https://github.com/vitejs/vite-plugin-react/pull/807.